pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -242,6 +242,22 @@ func CreateService(name string, service kobject.ServiceConfig, objects []runtime
 	servicePorts := ConfigServicePorts(name, service)
 	svc.Spec.Ports = servicePorts
 
+	// Configure service types
+	for key, value := range service.Annotations {
+		if key == "kompose.service.type" && (value == "NodePort" || value == "nodeport") {
+			svc.Spec.Type = "NodePort"
+			delete(service.Annotations, "kompose.service.type")
```

Comment:
I would keep original Annotation. It might be useful to have this information in future for keeping track where this value came from.